src/ssl.c

```diff
@@ -34562,44 +34543,125 @@ WOLFSSL_RSA *wolfSSL_PEM_read_bio_RSA_PUBKEY(WOLFSSL_BIO* bio,WOLFSSL_RSA** rsa,
 
 #endif /* defined(OPENSSL_EXTRA) || defined(OPENSSL_ALL) && !defined(NO_RSA) */
 
-/* Takes a public key and writes it out to a WOLFSSL_BIO
- * Returns WOLFSSL_SUCCESS or WOLFSSL_FAILURE
+/**
+ * The regen parameter determines if we will regenerate the DER buffer to
```

Comment:
No longer a region parameter?

Reply:
Yup. Refactored to do regen inside `WriteBioPUBKEY`.